jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -35,7 +35,7 @@ def agent_nodes = [
 
 pipeline {
     options {
-        timeout(time: 4, unit: 'HOURS')
+        timeout(time: 5, unit: 'HOURS')
```

Comment:
Strange! Why does OSD tests need 5 hours to run?

Reply:
Keeping it buffer for now Sayali, Wanted to check if the newly added tests are actually running. Currently the run is aborted due to timeout error.

Reply:
That timeout might be at the functional-test level

Reply:
There has been failing tests with pipeline timeout for OSD and components (Related to 2.11.1 release).
 https://build.ci.opensearch.org/blue/organizations/jenkins/integ-test-opensearch-dashboards/detail/integ-test-opensearch-dashboards/4572/pipeline, [https://build.ci.opensearch.org/blue/organizations/jenkins/integ-test-opensearch-dashboa[…]detail/integ-test-opensearch-dashboards/4548/pipeline/114](https://build.ci.opensearch.org/blue/organizations/jenkins/integ-test-opensearch-dashboards/detail/integ-test-opensearch-dashboards/4548/pipeline/114), want to allow increasing the time and run the tests.

Reply:
If this still timeouts, there needs a deep dive in https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/2.11/integtest.sh.